packages/channels/botpress-channel-web/src/views/web/side/style.scss

```diff
@@ -430,6 +430,35 @@
   }
 }
 
+.greetingScreen {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+
+  .getStartedBtn {
```

Comment:
You shouldn't use nesting in CSS Modules most of the time

Reply:
@emirotin I need it to rewrite styles